### PR TITLE
Merging Master back into Develop

### PR DESCRIPTION
**Pull request by aphelionz**

### CODE REVIEW

**Comment by aphelionz** [comment]
@pkdash @Maurier @hyi Can you review?

File: hs_core/views/resource_folder_hierarchy.py [hunk 1]
```diff
@@ -95,19 +103,20 @@ def data_store_structure(request):
...
         return HttpResponse(ex.stderr, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
-    except Exception as ex:
-        return HttpResponse(ex.message, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
     return_object = {'files': files,
                      'folders': store[0],
                      'can_be_public': resource.can_be_public_or_discoverable}
 
     if resource.resource_type == "CompositeResource":
+        # logger.debug("resource {}: begin CompositeResource handling".format(res_id))
```
Comment (hyi) [inline hs_core/views/resource_folder_hierarchy.py]: It'd be good if these debug comments can be removed.

File: hs_core/views/resource_folder_hierarchy.py [hunk 2]
```diff
@@ -95,19 +103,20 @@ def data_store_structure(request):
...
         spatial_coverage_dict = get_coverage_data_dict(resource)
         temporal_coverage_dict = get_coverage_data_dict(resource, coverage_type='temporal')
         return_object['spatial_coverage'] = spatial_coverage_dict
         return_object['temporal_coverage'] = temporal_coverage_dict
+        # logger.debug("resource {}: end CompositeResource handling".format(res_id))
```
Comment (hyi) [inline hs_core/views/resource_folder_hierarchy.py]: It'd be good if these debug comments can be removed.

**Comment by aphelionz** [comment]
@hyi removed

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Pull request merged**